src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/StaticHolderTypeTests.cs

```diff
@@ -823,5 +823,20 @@ internal class C2 : C1 {
 }
 ");
         }
+
+        [Fact]
+        public async Task CA1052NoDiagnosticRecords()
+        {
+            await new VerifyCS.Test
+            {
+                LanguageVersion = Microsoft.CodeAnalysis.CSharp.LanguageVersion.CSharp9,
```

Comment:
The fully qualified name is intentional here because these tests include both VB and C#.
Similar to what's currently done in https://github.com/dotnet/roslyn-analyzers/blob/7b28fef1af5060ce321f6c8761047581efc1f8d2/src/Roslyn.Diagnostics.Analyzers/UnitTests/DoNotCopyValueTests.cs